content/blog/2024-05-02-flash-expert.Rmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "A User's Perspective on the Updated FlaSH System"
+author: Tina Townes
+date: 2024-05-02
+tags:
+  - flash
+authors:
+  - tina
+heroImage: blog-thumb-flash-expert.png
+heroImageThumb: blog-lg-flash-expert.png
+summary: | 
+  A reflection of the recent changes to the FlaSH user experience by our quality assurance expert, Tina Townes. 
+output:
+  blogdown::html_page:
+    toc: true
+---
+
+<center>
+![**Fig 1a.** Revised FlaSH Dashoard](/blog/2024-05-02-flash-expert/new_dash.png)</center>
```

Comment:
some quick googling leads me to believe this might make the image clickable, which then opens the full image itself?
```suggestion
![**Fig 1a.** Revised FlaSH Dashoard](/blog/2024-05-02-flash-expert/new_dash.png)(/blog/2024-05-02-flash-expert/new_dash.png)</center>
```

or maybe
```
[![**Fig 1a.** Revised FlaSH Dashoard](/blog/2024-05-02-flash-expert/new_dash.png)](/blog/2024-05-02-flash-expert/new_dash.png)</center>
```